source/Weapon.h

```diff
@@ -174,14 +175,15 @@ class Weapon {
 	static const int HULL_DAMAGE = 1;
 	static const int HEAT_DAMAGE = 2;
 	static const int ION_DAMAGE = 3;
-	static const int DISRUPTION_DAMAGE = 4;
-	static const int SLOWING_DAMAGE = 5;
-	mutable double damage[6] = {0., 0., 0., 0., 0., 0.};
+	static const int FUEL_DAMAGE = 4; 
```

Comment:
Any reason why you chose to insert FUEL_DAMAGE between existing damage types, rather than at the end?

Reply:
I had inserted it here because I felt that it would make more sense to be grouped this way.